common/common-generator/src/main/java/com/pg85/otg/gen/resource/util/BerryBush.java

```diff
@@ -0,0 +1,86 @@
+package com.pg85.otg.gen.resource.util;
+
+import com.pg85.otg.constants.Constants;
+import com.pg85.otg.interfaces.IWorldGenRegion;
+import com.pg85.otg.util.materials.LocalMaterialData;
+import com.pg85.otg.util.materials.LocalMaterials;
+import com.pg85.otg.util.materials.MaterialProperties;
+import com.pg85.otg.util.materials.MaterialSet;
+import com.pg85.otg.util.minecraft.PlantType;
+
+import java.util.Random;
+
+public class BerryBush {
```

Comment:
> It makes much more sense making it so these options can be used by every plant resource 
It can be used by every Plant Resource 
`if (sparseOption != null && plant == PlantType.BerryBush){
            BerryBush.spawnBerryBushes(worldGenregion, rand, x, z, plant, frequency, minAltitude, maxAltitude, sourceBlocks, sparseOption);
			return;
        }`
I just called the file BerryBush because it generates the spawn clusters resembling the BerryBush spawn clusters.